app/routes/view-application.js

```diff
@@ -103,7 +103,7 @@ module.exports = {
             ? { text: 'Select both checkboxes' }
             : undefined
         },
-        subStatus,
+        subStatus: upperFirstLetter(subStatus.toLowerCase()),
```

Comment:
it would be slightly better if upper casing was done somewhere here:

<img width="474" alt="image" src="https://github.com/DEFRA/ffc-ahwr-backoffice/assets/118533775/9f11e7ce-4fd1-4368-8b48-83136fd37d0a">

we could also add rbacEnabled to the properties that we log